avogadro/qtplugins/surfaces/surfaces.h

```diff
@@ -94,7 +95,9 @@ private slots:
 
   Core::Cube* m_cube = nullptr;
   std::vector<Core::Cube*> m_cubes;
-  QFutureWatcher<void> m_cubesWatcher;
+  QFutureWatcher<void> m_performEDTStepWatcher;
```

Comment:
I would add comments on what each of these do. Otherwise, you find yourself coming back 2 months or 2 years later and have to re-read your code.

I tend to *over-comment* things. 😄

Reply:
Done!